examples/multi-cluster/kustomization.yaml

```diff
@@ -0,0 +1,20 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+namespace: <namespace here>
+kind: Kustomization
+nameSuffix: -cluster1
+commonLabels:
+  app: cockroachdb-cluster1
+configMapGenerator:
+- envs:
+  - config/cockroach
```

Comment:
This doesn't build for me: there's no `config/` directory

aside (thought for the future): maybe we should have some CI to ensure examples always build